### PR TITLE
radare2: bump version to 1.1.0

### PR DESCRIPTION
**Pull request by return**
Tested on both x86 and x86_64, but not x86_gcc2. This patch fixes issues with the login_pty symbol when running radare2.

### CODE REVIEW

File: dev-util/radare2/radare2-1.1.0.recipe [hunk 1]
```diff
@@ -8,16 +8,18 @@ COPYRIGHT="2009-2015 nibble, pancake"
 LICENSE="GNU GPL v3
 	GNU LGPL v3"
 REVISION="1"
-SOURCE_URI="http://radare.org/get/radare2-$portVersion.tar.xz"
-CHECKSUM_SHA256="024adba5255f12e58c2c1a5e2263fada75aad6e71b082461dea4a2b94b29df32"
+SOURCE_URI="http://cloud.radare.org/get/1.1.0/radare2-$portVersion.tar.gz"
```
Comment (pulkomandy) [inline dev-util/radare2/radare2-1.1.0.recipe]: Please use $portVersion rather than 1.1.0 here.

File: dev-util/radare2/radare2-1.1.0.recipe [hunk 2]
```diff
@@ -93,11 +98,6 @@ INSTALL()
 {
 	make install
 	cp libr/*/libr_*.so $libDir
-
-	mkdir -p `dirname $docDir`
-	mv $prefix/share/doc $docDir
-	rm -rf $prefix/share
```
Comment (pulkomandy) [inline dev-util/radare2/radare2-1.1.0.recipe]: Are the docs properly installed in $docDir now (ie is that fixed upstream?)

Comment (return) [inline dev-util/radare2/radare2-1.1.0.recipe]: I don't think this is fixed upstream, Line 98 causes the build to fail in 1.1.0  when moving the docs to $docDir.

Comment (return) [inline dev-util/radare2/radare2-1.1.0.recipe]: But moving the directory from $prefix/data/doc to $docDir puts the documentation under documentation/packages/radare2/radare2/  

Is this correct documentation directory to place the html/markdown files?

**Pull request merged**